include/dem/particle_particle_contact_force.h

```diff
@@ -1129,131 +1045,300 @@ class ParticleParticleContactForce
       particle_one_properties[PropertiesIndex::type];
     const unsigned int particle_two_type =
       particle_two_properties[PropertiesIndex::type];
+    const unsigned int property_index =
+      vec_particle_type_index(particle_one_type, particle_two_type);
 
-    const double radius_times_overlap_sqrt =
-      sqrt(this->effective_radius * normal_overlap);
-    const double model_parameter_sn =
-      2.0 *
-      this->effective_youngs_modulus[vec_particle_type_index(
-        particle_one_type, particle_two_type)] *
-      radius_times_overlap_sqrt;
-    double model_parameter_st =
-      8.0 *
-      this->effective_shear_modulus[vec_particle_type_index(
-        particle_one_type, particle_two_type)] *
-      radius_times_overlap_sqrt;
-
-    // Calculation of normal and tangential spring and dashpot constants
-    // using particle properties
-    double normal_spring_constant = 0.66665 * model_parameter_sn;
-    double normal_damping_constant =
-      -1.8257 *
-      this->model_parameter_beta[vec_particle_type_index(particle_one_type,
-                                                         particle_two_type)] *
-      sqrt(model_parameter_sn * this->effective_mass);
-    double tangential_spring_constant =
-      8.0 *
-        this->effective_shear_modulus[vec_particle_type_index(
-          particle_one_type, particle_two_type)] *
-        radius_times_overlap_sqrt +
-      DBL_MIN;
-    double tangential_damping_constant =
-      normal_damping_constant * sqrt(model_parameter_st / model_parameter_sn);
-
-    // Calculation of normal force. The cohesive term is computed here.
-    const double cohesive_term =
-      2. * M_PI * effective_radius *
-      this->effective_surface_energy[vec_particle_type_index(
-        particle_one_type, particle_two_type)];
-    double normal_force_norm =
-      normal_spring_constant * normal_overlap +
-      normal_damping_constant * normal_relative_velocity_value;
-
-    // Calculation of tangential force. Since we need damping tangential force
-    // in the gross sliding again, we define it as a separate variable
-    Tensor<1, 3> damping_tangential_force =
-      tangential_damping_constant * tangential_relative_velocity;
-    tangential_force =
-      (tangential_spring_constant * contact_info.tangential_overlap) +
-      damping_tangential_force;
+    const double F_so = 6.283185307179586 * // 2. * M_PI
+                        this->effective_radius *
+                        this->effective_surface_energy.at(property_index);
 
-    double coulomb_threshold =
-      this->effective_coefficient_of_friction[vec_particle_type_index(
-        particle_one_type, particle_two_type)] *
-      normal_force_norm;
+    const double delta_0 = -std::sqrt(
+      this->effective_hamaker_constant.at(property_index) / (6. * F_so));
```

Comment:
Not supposed to be 
`-std::sqrt(
      -this->effective_hamaker_constant.at(property_index) *this->effective_radius  / (6. * F_so));`
?

Reply:
I just pushed something. 

F_po is always negative, so no. But you were right haha.

Reply:
Oh ok, what about the effective radius?

Reply:
you were also right haha